CoreOnly/Sources/Firebase.h

```diff
@@ -85,18 +85,6 @@ FirebaseAnalytics dependency to your project to ensure Messaging works as intend
     #endif
   #endif
 
-  #if __has_include(<FirebaseMLCommon/FirebaseMLCommon.h>)
-    #import <FirebaseMLCommon/FirebaseMLCommon.h>
-  #endif
-
-  #if __has_include(<FirebaseMLModelInterpreter/FirebaseMLModelInterpreter.h>)
-    #import <FirebaseMLModelInterpreter/FirebaseMLModelInterpreter.h>
-  #endif
-
-  #if __has_include(<FirebaseMLVision/FirebaseMLVision.h>)
-    #import <FirebaseMLVision/FirebaseMLVision.h>
-  #endif
-
```

Comment:
This is a breaking change - even though it's deprecated, anyone using the frameworks currently would fail to find them anymore. This portion need to wait until Firebase 8